coderadar-core/src/main/java/io/reflectoring/coderadar/vcs/service/DeleteRepositoryService.java

```diff
@@ -0,0 +1,23 @@
+package io.reflectoring.coderadar.vcs.service;
+
+import io.reflectoring.coderadar.vcs.port.driven.DeleteRepositoryPort;
+import io.reflectoring.coderadar.vcs.port.driver.DeleteRepositoryUseCase;
+import java.io.IOException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DeleteRepositoryService implements DeleteRepositoryUseCase {
```

Comment:
Rename to `DeleteLocalRepositoryService`

Reply:
Deleted